src/index.js

```diff
@@ -221,6 +228,45 @@ io.on('connection', function(socket) {
     }
   });
 
+  socket.on('cl_buy_life', async (done) => {
```

Comment:
Change 'cl_buy_life' to 'cl_store_buy_life'.

Reply:
done